kie-pmml/src/main/java/org/kie/pmml/pmml_4_2/model/ExternalBeanDefinition.java

```
@@ -1,14 +1,25 @@
 package org.kie.pmml.pmml_4_2.model;
 
+import org.kie.pmml.pmml_4_2.PMMLError;
+
 public class ExternalBeanDefinition {
 
     private String beanPackageName;
     private String beanName;
+    public static final String DEFAULT_BEAN_PKG = "org.kie.pmml.externalbean";
 
-    public ExternalBeanDefinition(String beanInfo) {
+    public ExternalBeanDefinition(String beanInfo) throws Exception {
+    	if (beanInfo == null || beanInfo.isEmpty()) {
+    		throw new Exception("External type name missing");
```

Please don't use general Exception type for throwing exceptions. This makes handling of such exceptions very hard, because the type of the exception is hidden. For this, IllegalArgumentException can be used - it makes the exception specific - Illegal value for beanInfo argument was provided. 

Minor edit: please reformat the throw line, it is off to the right by 4 spaces.